resources/local/config.toml

```diff
@@ -236,20 +239,39 @@ qps_limit = 100
 # Maximum number of bytes to accept in a single request body.
 max_body_bytes = 2_621_440
 
-# Address to which the speculative execution endpoint binds to.
-speculative_execution_address = "0.0.0.0:7778"
+
+# ===================================================================
+# Configuration options for the speculative execution RPC HTTP server
```

Comment:
It's JSON RPC server. I don't really understand whether we use these terms correctly but we seem to use that to differentiate.

Reply:
Here and everywhere else.

Reply:
Done.